src/context/wishListContext.js

```diff
@@ -0,0 +1,83 @@
+import { createContext, useContext, useReducer } from "react";
+import { useFetchApi } from "../hooks/useFetchApi";
+import { useAuth } from "./";
+import { initialSharedState } from "../reducers/constants";
+import { sharedReducer } from "../reducers/reducerFunctions/sharedReducer";
+import jwt from "jsonwebtoken";
+import axios from "axios";
+
+const WishListContext = createContext();
+
+const WishListProvider = ({ children }) => {
+  const [wishlistState, wishlistDispatch] = useReducer(
+    sharedReducer,
+    initialSharedState
+  );
+  const apiData = {
+    api: "/api/user/wishlist",
+    property: "wishList",
+  };
+  const { userAuthState } = useAuth();
+  const { authToken } = userAuthState;
+
+  const { state } = useFetchApi(apiData, {
+    headers: {
```

Comment:
This `headers` configuration is getting repeated three times, so you can create a variable and then can use that. Like

```javascript
const config = {
         headers: {
            authorization: authToken,
          },
}
```

And then, can use like
```javascript
await axiost.get(api, config)
```

Reply:
will check this